plugins/proofpoint_tap/plugin.spec.yaml

```
@@ -627,6 +627,190 @@ types:
       type: "[]urls"
       description: "Urls"
       required: false
+  platform:
+    name:
+      title: Name
+      description: Name of platform
+      type: string
+      required: false
+    os:
+      title: OS
+      description: Operating system
+      type: string
+      required: false
+    version:
+      title: Version
+      description: Version of operating system
+      type: string
+      required: false
+  evidence_type:
+    sha256:
+      title: SHA256
+      description: The SHA256 hash of the attachment's contents
+      type: string
+      required: false
+    blacklisted:
+      title: Blacklisted
+      description: Optional, whether the file was blocklisted
+      type: integer
+      required: false
+    md5:
+      title: MD5
+      description: Optional, the MD5 sum of the attachment's contents
+      type: string
+      required: false
+    offset:
+      title: Offset
+      description: Optional, the offset in bytes where the malicious content was found
+      type: integer
+      required: false
+    rule:
+      title: Rule
+      description: Optional, the name of the static rule inside the sandbox which located the malicious content
+      type: string
+      required: false
+    size:
+      title: Size
+      description: Optional, the size in bytes of the attachment's contents
+      type: integer
+      required: false
+    action:
+      title: Action
+      description: Whether the cookie was set or deleted
+      type: string
+      required: false
+    domain:
+      title: Domain
+      description: Which domain set the cookie
+      type: string
+      required: false
+    key:
+      title: Key
+      description: The name of the cookie being set or deleted
+      type: string
+      required: false
+    value:
+      title: Value
+      description: Optional, content of the cookie being set
+      type: string
+      required: false
+    host:
+      title: Host
+      description: The hostname being resolved
+      type: string
+      required: false
+    cnames:
+      title: Cnames
+      description: Optional, an array of cnames which were associated with the hostname
+      type: "[]string"
+      required: false
+    ips:
+      title: IPs
```

```suggestion
      title: Resolved IP Addresses
```
Seems like a more descriptive title based off the description